core/domain/fs_services_test.py

```diff
@@ -27,10 +27,10 @@ def test_get_exploration_file_system_with_dev_mode_enabled(self):
         with self.swap(constants, 'DEV_MODE', True):
             file_system = fs_services.get_exploration_file_system_class()
             self.assertIsInstance(
-                file_system('exploration_id'), fs_domain.ExplorationFileSystem)
+                file_system('entity_id', None), fs_domain.EntityFileSystem)
```

Comment:
This should probably change to not use None, ditto below.

Reply:
Done